projects/angular/src/data/tree-view/tree-focus-manager.service.ts

```diff
@@ -101,6 +101,7 @@ export class TreeFocusManagerService<T> {
     } else if (selfIndex === siblings.length - 1) {
       return this.findNextFocusable(model.parent);
     }
+
```

Comment:
```suggestion
```

The only change in this file is adding a blank line.

Reply:
removed